technique/muon/background_plot.py

```diff
@@ -118,6 +129,32 @@ def _first_point(self):
                 for point in first_point[1:]:
                     self.data.append([point, ])
 
+    def load_data_from_save(self):
+        """
+        Imports data from the save file
+        """
+        # Copy nested list structure from first point
+        loaded_data = [list() for x in range(len(self.data))]
+        loaded_data_x = []
+
+        with open(self._save_file, 'r') as csvfile:
+            file_without_header = filter(lambda row: row if not row.startswith('#') else '', csvfile)
+            reader = csv.reader(file_without_header)
+
+            for row in reader:
+                loaded_data_x.append(datetime.fromisoformat(row[0]))
+
+                for dataset, restored_dataset in zip(loaded_data, row[1:]):
```

Comment:
A couple of extra comments in this area might be helpful - e.g. this loop is reasonably complex